From c6880daeb7042f88d98c165d33a797dc8081c251 Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Fri, 28 Jul 2017 15:36:07 +0800
Subject: [PATCH] soc: rockchip: pvtm: fix undefined reference to
 rockchip_get_pvtm_value

This patch fixed compilation error when compiled rockchip_linux_defconfig
without CONFIG_ROCKCHIP_PVTM

Change-Id: Icb43485a9ce8dca0e40507b370487b89617813c4
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 include/linux/soc/rockchip/pvtm.h | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/include/linux/soc/rockchip/pvtm.h b/include/linux/soc/rockchip/pvtm.h
index ee54d6228c75..11f99dfa7b0c 100644
--- a/include/linux/soc/rockchip/pvtm.h
+++ b/include/linux/soc/rockchip/pvtm.h
@@ -1,7 +1,15 @@
 #ifndef __SOC_ROCKCHIP_PVTM_H
 #define __SOC_ROCKCHIP_PVTM_H
 
+#ifdef CONFIG_ROCKCHIP_PVTM
 u32 rockchip_get_pvtm_value(unsigned int ch, unsigned int sub_ch,
 			    unsigned int time_us);
+#else
+static inline u32 rockchip_get_pvtm_value(unsigned int ch, unsigned int sub_ch,
+					  unsigned int time_us)
+{
+	return 0;
+}
+#endif
 
 #endif /* __SOC_ROCKCHIP_PVTM_H */
-- 
2.35.3

